tetris.py

```diff
@@ -177,9 +228,46 @@ def draw_board(canvas):
         x_gap += square_size
 
 
+def get_high_scores():
+    index = 0
+    treeview.delete(*treeview.get_children())
+    for high_score_dict in json_dict['high_scores']:
+        if index == 0:
+            tag = 'odd_row'
+        else:
+            tag = 'even_row'
+        treeview.insert(parent="", index="end", tags=tag, values=(high_score_dict['Time'], high_score_dict['Game Speed'], high_score_dict['Score']))
+        index += 1
+        index %= 2
+    treeview.tag_configure('odd_row', background=D_GREY)
+    treeview.tag_configure('even_row', background='black')
+
+
+def display_high_scores():
+    get_high_scores()
+    try:
+        game_canvas.pack_info()
+        game_canvas.pack_forget()
+        if tetris_control.game.status == GameStatus.in_progress:
+            tetris_control.pause_game()
+        treeview.pack(side='top', fill='both', expand=True)
+    except tkinter.TclError:
+        treeview.pack_forget()
+        game_canvas.pack(fill='both', expand=True)
+
+
+def clear_high_scores():
```

Comment:
It's too easy to clear the high scores accidentally. Maybe ask the user whether they really want to clear them, similar to starting a new game?

Reply:
Added a dialogue window for clear list

Reply:
Can't see the code for it?

Reply:
Didn't push :( Damn it!